sandbox/my_sod_shock_stube_solver/my_cese_solver.py

```diff
@@ -0,0 +1,186 @@
+"""
+Sod Shock Tube Problem -- CESE solver
+Editor: You-Hao Chang
+
+Journal of Computational Physics 119, 295-324 (1995)
+The Method of Space-Time Conservation Element and Solution
+Element -- A New Approach for Solving the Navier-Stokes and 
+Eluer Equations
+
+Sod tube condition: 
+    x range: 101 points btw -0.505 ~ 0.495
+    two region: left region, -0.505 ~ -0.005 (51 points)
+                right region, 0.005 ~  0.495 (50 points)
+"""
+
+import matplotlib
+matplotlib.use('TKAgg')
+
+import math
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+
+# initialization
+it = 100            # number of iterations
+nx = it + 1         # number of points (x-axis)
+dt = 0.4 * 10**(-2)
+dx = 0.1 * 10**(-1)
+ga = 1.4            #gamma = Cp/Cv
+
+rhol = 1.0
+pl   = 1.0
+ul   = 0.0
+rhor = 0.125
+pr   = 0.1
+ur   = 0.0
+
+mtx_q  = [[0. for dummy_col in range(nx)] for dummy_row in range(3)]
```

Comment:
This is very inefficient in runtime and memory usage.  In Python we should use as much as possible numpy ndarray fo numercal calculation.  Although element-wise operations with ndarray is still very slow in Python, later we can migrate the ndarray code to Cython to achieve the speed comparable to C.  (One familiar with Cython can just write Cython code at the very beginning.)